projects/vdk-plugins/airflow-provider-vdk/vdk_provider/sensors/vdk.py

```diff
@@ -45,4 +50,30 @@ def poke(self, context: Dict[Any, Any]) -> bool:
         :param context: Airflow context passed through the DAG
         :return: True if some job status condition is met; False otherwise
         """
-        pass
+        vdk_hook = VDKHook(self.conn_id, self.job_name, self.team_name)
+
+        job_execution = vdk_hook.get_job_execution_status(self.job_execution_id)
+        job_status = job_execution.status
+
+        log.info(
+            f"Current status of job execution {self.job_execution_id} is: {job_status}."
+        )
+
+        if job_status == "succeeded":
+            log.info(f"Job status: {job_execution}")
+            log.info(
+                f"Job logs: {vdk_hook.get_job_execution_log(self.job_execution_id)}"
+            )
+
+            return True
+        elif job_status == "cancelled" or job_status == "skipped":
```

Comment:
Can we get these statuses from vdk-control-service-api as constants?

Reply:
I tried to but this is how they're listed in the DataJobExecution object:
```
@status.setter
    def status(self, status):
        """Sets the status of this DataJobExecution.

        The current status  # noqa: E501

        :param status: The status of this DataJobExecution.  # noqa: E501
        :type: str
        """
        allowed_values = ["submitted", "running", "succeeded", "cancelled", "skipped", "user_error", "platform_error"]  # noqa: E501
        if self.local_vars_configuration.client_side_validation and status not in allowed_values:  # noqa: E501
            raise ValueError(
                "Invalid value for `status` ({0}), must be one of {1}"  # noqa: E501
                .format(status, allowed_values)
            )

        self._status = status
```

Reply:
Ok, let's define them in the hook in order to reuse them in both operators.

Reply:
noted